build/terraform/gke/module.tf

```diff
@@ -41,11 +37,30 @@ variable "machine_type" {
   default = "n1-standard-4"
 }
 
+variable "name" {
+  default = "agones-terraform-example"
```

Comment:
Since this is now in our build scripts, should this be something like `agones-tf-cluster` ? So it's similar to the other cluster name, which is `test-cluster`. This is no longer an example, seems like a strange name.

What do you think? That was the best name I could come up with.